public/data/Pritam0077.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Pritam Panda",
+  "type": "personal",
+  "bio": "Web Developer and Open Source Enthusiast!",
+  "avatar": "https://github.com/Pritam0077.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Pritam0077",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/PritamP09066621",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Coding",
+      "date": "December 2020",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing"
+      "url": "http://linkfree.eddiehub.org/Pritam0077"
```

Comment:
Why the linkfree url here?

Reply:
it was given in the documentation to add the URL like this

Reply:
Only wonder why the linking to the same page the user currently is.  But good for me.